Source/CombatExtended/CombatExtended/Jobs/JobDriver_Hunt.cs

```diff
@@ -68,10 +68,18 @@ protected override IEnumerable<Toil> MakeNewToils()
             yield return Toils_Jump.JumpIfTargetDespawnedOrNull(VictimInd, startCollectCorpse);
 
             var startExecuteDowned = Toils_Goto.GotoThing(VictimInd, PathEndMode.Touch).JumpIfDespawnedOrNull(VictimInd, startCollectCorpse);
+
+            var startExecuteDownedRanged = Toils_Combat.CastVerb(VictimInd);
```

Comment:
Isn't this redundant? If the jump to startExecuteDowned fails to trigger it will proceed to shoot it anyways.

Reply:
Each toil is always called, so without startExecuteDownedRanged the pawn will always try to approach a downed target. This forces the pawn to fire at the downed target if the toil condition is met.

Reply:
>if the toil condition is met

Which doesn't happen, hence my point.